### PR TITLE
Sensitivity labels in OWA are GA, Message Classification section needs to only cover Exchange Server

### PR DESCRIPTION
**Pull request by Esaggese**
…ange Server

Eliminated all references to using this solution for OWA in Office 365, since it is only useful for backwards compatibility with Exchange server.

### CODE REVIEW

**Comment by PRMerger9** [comment]
@Esaggese : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: Azure-RMSDocs/rms-client/client-admin-guide-customizations.md [hunk 1]
```diff
@@ -979,11 +979,11 @@ Set the logging level to one of the following values:
...
+## Integration with the legacy Exchange message classification
 
-Outlook on the web now supports built-in labeling for Exchange Online, which is the recommended method to label emails in Outlook on the web. However, if you're not yet using sensitivity labels that are published from the Office 365 Security & Compliance Center, Microsoft 365 security center, or Microsoft compliance center, you can use Exchange message classification to extend Azure Information Protection labels to your mobile users when they use Outlook on the web. You can also use this method for Exchange Server. 
+Outlook on the web now supports built-in labeling for Exchange Online, which is the recommended method to label emails in Outlook on the web. However, if you need to label emails in OWA and are using Exchange Server, which doesn't yet support sensitivity labels, you can use Exchange message classification to extend Azure Information Protection labels to Outlook on the web.
 
-Outlook Mobile does not support Exchange message classification.
+Please note that Outlook Mobile does not support Exchange message classification.
```
Comment (mlottner) [inline Azure-RMSDocs/rms-client/client-admin-guide-customizations.md:986]: Outlook Mobile does not support Exchange message classification.

**Comment by PRMerger12** [comment]
@Esaggese : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by mlottner** [comment]
#sign-off

**Pull request merged**